src/nodes.coffee

```diff
@@ -2233,6 +2244,42 @@ exports.Parens = class Parens extends Base
 # string concatenation inside.
 
 exports.StringWithInterpolations = class StringWithInterpolations extends Parens
+  unwrap: -> this
```

Comment:
`"#{2}"` isn’t pointless, as it allows you to cast a value as a string. I know I’ve written the code `"" + foo` on occasion. It does come up.